build/import/Versions.props

```diff
@@ -1,7 +1,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <ProjectSystemVersion>16.8.1</ProjectSystemVersion>
+    <ProjectSystemVersion>16.10.0</ProjectSystemVersion>
```

Comment:
What is this property used for? Does it matter that we skipped a 16.9 value?

Reply:
Based on the errors in the build, we enforce that the specified version of Visual Studio is installed. If that is all we use it for, we should consider renaming it.

Reply:
We used to have `VisualStudioVersion` and `ProjectSystemVersion` but they used to match always so we decided to keep them the same. 16.9 was skipped by mistake but it only affects our package names so just mildly annoying but not a big impact. The build machines aren't always updated with the latest VS version so it might be useful to bring the `VisualStudioVersion` back.